tests/jobs/test_backfill_job.py

```diff
@@ -608,11 +609,44 @@ def test_backfill_override_conf(self, dag_maker):
             dr = wrapped_task_instances_for_dag_run.call_args_list[0][0][0]
             assert dr.conf == {"a": 1}
 
+    def test_backfill_skip_active_scheduled_dagrun(self, dag_maker):
+        dag = self._get_dummy_dag(
+            dag_maker,
+            dag_id="test_backfill_skip_active_scheduled_dagrun",
+            task_id="test_backfill_skip_active_scheduled_dagrun-1",
+        )
+        dag_maker.create_dagrun(
+            run_type=DagRunType.SCHEDULED,
+            state=State.RUNNING,
+        )
+
+        executor = MockExecutor()
+
+        job = BackfillJob(
+            dag=dag,
+            executor=executor,
+            start_date=DEFAULT_DATE,
+            end_date=DEFAULT_DATE + datetime.timedelta(days=2),
+        )
+        with patch.object(job.log, 'error') as mock_error:
```

Comment:
Don't need to mock this, you can use pytest’s `caplog` fixture.

Reply:
TIL, thanks for the tip!